### DISCUSSION TITLE
Pregunta sobre entrega 1 proyecto semestre

### DISCUSSION THREAD

**Discussion by HernanAC** [rramosp/ai4eng.v1#234] [00 - General]
Ya realicé la inscripción del proyecto mediante el Google Forms, y subí a mi repositorio ( https://github.com/HernanAC/TaiwaneseBankruptcyPrediction ) el archivo "PROYECTO_ENTREGA1.pdf". 

¿Con esto es suficiente, o sería necesario realizar un paso más para completar la entrega 1?

Muchas gracias.

**Comment by rramosp** [comment]
así es suficiente ... cada miembro del equipo debe de tener su github y
registrar el proyecto, si no, sólo se te contará la entrega a ti.

On Wed, Mar 1, 2023 at 9:53 AM HernanAC ***@***.***> wrote:

> Ya realicé la inscripción del proyecto mediante el Google Forms, y subí a
> mi repositorio ( https://github.com/HernanAC/TaiwaneseBankruptcyPrediction
> ) el archivo "PROYECTO_ENTREGA1.pdf".
>
> ¿Con esto es suficiente, o sería necesario realizar un paso más para
> completar la entrega 1?
>
> Muchas gracias.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/234>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIIBE7T23J43J56VCELWZ5PGZANCNFSM6AAAAAAVME5L44>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by HernanAC** [reply]
Listo profesor, muchas gracias.
